packages/communication-react/review/communication-react.api.md

```diff
@@ -2025,18 +2053,9 @@ export interface VideoGalleryProps {
     dominantSpeakers?: string[];
     layout?: 'default' | 'floatingLocalVideo';
     localParticipant: VideoGalleryLocalParticipant;
-    localVideoViewOption?: VideoStreamOptions;
-    onCreateLocalStreamView?: (options?: VideoStreamOptions) => Promise<void>;
-    onCreateRemoteStreamView?: (userId: string, options?: VideoStreamOptions) => Promise<void>;
-    onDisposeLocalStreamView?: () => void;
-    // (undocumented)
-    onDisposeRemoteStreamView?: (userId: string) => Promise<void>;
-    onRenderAvatar?: OnRenderAvatarCallback;
-    onRenderLocalVideoTile?: (localParticipant: VideoGalleryLocalParticipant) => JSX.Element;
```

Comment:
For handlers - we should not remove them from videoGallery props, it's still here and pass down to the VideoTile